src/graph_test.cc

```diff
@@ -282,6 +282,54 @@ TEST_F(GraphTest, InputsCollector) {
   EXPECT_EQ("out3", inputs[4]);
 }
 
+TEST_F(GraphTest, InEdgeCollector) {
```

Comment:
You added this test between InputsCollector and InputsCollectorWithEscapes, can you move it past the latter instead?